src/partialBindTo.ts

```diff
@@ -0,0 +1,74 @@
+import { purry } from './purry';
+
+type RemovePrefix<
+  P extends ReadonlyArray<unknown>,
+  T extends ReadonlyArray<unknown>,
+> = P extends readonly [...T, ...infer U]
+  ? U
+  : ["ERROR(partialBindTo): Data doesn't match function signature."];
```

Comment:
I'm afraid that engineers that don't know Remeda well might find the error message confusing, let's add more context:
```suggestion
  : ["RemedaTypeError(partialBindTo): Data doesn't match function signature."];
```

Reply:
the only other place where we have a similar type error message is with `fromPairs` i think, and that one uses ERROR—though maybe it'd be nice to audit the library and check other places where `never` is used and change that

Reply:
anyway, will change